api/pages/issue/trends.py

```diff
@@ -324,7 +325,8 @@ def run(API, environ, indata, session):
             body = query
         )
     no_closers_before = res['aggregations']['closer']['value']
-    
+    issuesRatioBefore = round(float(no_issues_closed_before) / no_issues_created_before, 2)
```

Comment:
Hey @amunz 
```suggestion
    issues_ratio_before = round(float(no_issues_closed_before) / no_issues_created_before, 2)§
```
Let's make it python style. ;)